ci/scrub-kicad-project-files

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# prototype-kicad-library-ci
+
+# File: /ci/scrub-kicad-project-files
+# Description: prototype-kicad-library-ci repository KiCad project file scrubbing script.
+
+function error()
+{
+    local -r message="$1"
+
+    ( >&2 echo "$mnemonic: $message" )
+}
+
+function abort()
+{
+    if [[ "$#" -gt 0 ]]; then
+        local -r message="$1"
+
+        error "$message, aborting"
+    fi
+
+    exit 1
+}
+
+function display_help_text()
+{
+    echo "NAME"
+    echo "    $mnemonic - Ensure KiCad project file's 'update' and 'last_client' fields are empty."
```

Comment:
Please use a column limit of 90 (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide) for consistency with other project scripts (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/42eb3668c6d55fc758e4ab2d8749f7c901471a3b/ci/analyze#L36-L37).
```suggestion
    echo "    $mnemonic - Ensure KiCad project file's 'update' and 'last_client' fields"
    echo "        are empty."
```